src/use-mutation.ts

```diff
@@ -255,6 +255,12 @@ export function useMutation<
           )) {
             // TODO: find a way to pass a source of the invalidation, could be a symbol associated with the mutation, the parameters
             store.invalidate(entry)
+            // auto refresh of the active queries
+            if (entry.active) {
+              // reset any pending request
+              entry.pending = null
```

Comment:
We can remove this since it's done by invalidate when cancelling the query
```suggestion
```

Reply:
Indeed, done.